src/WebJobs.Script.WebHost/Security/KeyManagement/BlobStorageSecretsRepository.cs

```diff
@@ -17,7 +17,7 @@ namespace Microsoft.Azure.WebJobs.Script.WebHost
     /// <summary>
     /// An <see cref="ISecretsRepository"/> implementation that uses Azure blob storage as the backing store.
     /// </summary>
-    public sealed class BlobStorageSecretsRepository : ISecretsRepository, IDisposable
```

Comment:
Can we `BlobStorageSecretsMigrationRepository` a simple decorator and avoid modifying this class?

Reply:
What do you mean by "decorator"? Something like:
public class BlobStorageSecretsMigrationRepository 
{
     private BlobStorageSecretsRepository _repo;
     pubic BlobStorageSecretsMigrationRepository(BlobStorageSecretsRepository repo)
     {
          _repo = repo;
     }
}